maintainers/maintainer-list.nix

```diff
@@ -1647,6 +1647,7 @@
     email = "me@artem.ist";
     github = "artemist";
     githubId = 1226638;
+    matrix = "@artemist:mildlyfunctional.gay";
```

Comment:
Looks like this was added by accident?

```suggestion
```

Reply:
not quite... it was added by artemis, who has been collaborating with me on this project, on our private staging branch, but I suppose this diff does remove the attribution for that line, which is probably important for the maintainers file.